crates/cheatable-starknet/Cargo.toml

```diff
@@ -26,5 +26,8 @@ starknet = "0.4.0"
 regex = "1.9.1"
 assert_fs = "1.0.13"
 
+[dev-dependencies]
+indoc.workspace = true
```

Comment:
Why was this added?

Reply:
Needed to use indoc in cheatable-starknet so I decided to hoist this dep here